src/components/text-editor/prosemirror-adapter/plugins/image/node.ts

```diff
@@ -123,7 +123,10 @@ function createImageNodeSpec(language: Languages): NodeSpec {
                         src: dom.getAttribute('src') || '',
                         alt: dom.getAttribute('alt') || 'file',
                         width: dom.style.width || '',
-                        maxWidth: dom.style.maxWidth || '100%',
+                        maxWidth:
+                            dom.style.maxWidth === 'none'
```

Comment:
When pasting image tags, in some scenarios the maxWidth was set to 'none'. In these scenarios, I believe we want to set it to 100 % instead.